src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -495,13 +503,182 @@ xclLoadXclBin(const xclBin *buffer)
   return ret;
 }
 
+#if defined(XRT_ENABLE_LIBDFX)
+namespace libdfx {
+
+static int
+libdfxHelper(std::shared_ptr<xrt_core::device> core_dev, std::string& dtbo_path, int& fd)
+{
+  uint32_t slot_id = 0;
+  static const std::string dtbo_dir_path = "/configfs/device-tree/overlays/";
+
+  // get dtbo_path of slot '0'
+  // TODO: read slot_id from xclbin
```

Comment:
@rozumx this is for future, when we support multi slot we need to get information about which slot this xclbin needs to be loaded so that we can remove already loaded bitstream/xclbin from that particular slot. The slot id info can be either coming from xclbin or as an argument

Reply:
@rozumx, added comment that explains things better. Please review,